gcloud-java-core/pom.xml

```diff
@@ -90,4 +90,102 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-site-plugin</artifactId>
+        <version>3.4</version>
+        <configuration>
+          <skipDeploy>true</skipDeploy>
+          <reportPlugins>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-changelog-plugin</artifactId>
+              <version>2.3</version>
+            </plugin>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-project-info-reports-plugin</artifactId>
+              <version>2.8</version>
+              <reportSets>
+                <reportSet>
+                  <reports>
+                    <report>index</report>
+                    <report>dependency-info</report>
+                    <report>dependencies</report>
+                    <report>dependency-convergence</report>
+                    <report>project-team</report>
+                    <report>mailing-list</report>
+                    <report>cim</report>
+                    <report>issue-tracking</report>
+                    <report>license</report>
+                    <report>scm</report>
+                    <report>dependency-management</report>
+                    <report>distribution-management</report>
+                    <report>summary</report>
+                    <report>modules</report>
+                  </reports>
+                </reportSet>
+              </reportSets>
+              <configuration>
```

Comment:
I took a look at the maven link explaining `combine.self="override"` before merging, and my understanding was that it would erase parent configurations we wanted in the child modules.  I don't think `combine.self="append"` would have helped either (to avoid copying `maven site` configurations), since the default strategy is to "merge", which should have given the same behavior as "append".